src/init.cc

```diff
@@ -353,7 +353,11 @@ static ncclResult_t commFree(ncclComm_t comm) {
   struct ncclProf *prof, *prof_seq;
   prof = (struct ncclProf*)malloc(sizeof(struct ncclProf)*MAXCHANNELS*PROFILE_NUM_LAUNCHES);
   CUDACHECK(hipMemcpy(prof, comm->devComm->devProf, sizeof(struct ncclProf)*MAXCHANNELS*PROFILE_NUM_LAUNCHES, hipMemcpyDeviceToHost));
-  #define VEGA_GPU_RTC_FREQUENCY 2.5E7
+#if defined(__gfx940__)
```

Comment:
compilation time detection doesn't work on host side. need to query device property